source/dofs/dof_handler_policy.cc

```diff
@@ -71,6 +71,15 @@ namespace internal
 
       namespace
       {
+        /**
+         * We'll use this constant to mark unnumbered degrees of freedom
+         * during their enumeration in multiple parts of the code.
+         * This constant is necessary to distinguish between valid and
+         * invalid degrees of freedom.
+         */
+        static const types::global_dof_index enumeration_dof_index =
```

Comment:
since you're already in an anonymous namespace, the `static` is not actually necessary here